### PR TITLE
Link incidents to components

### PR DESCRIPTION
**Pull request by jbrooksuk**
Linking an incident to a component has no effect on the component status yet.

### CODE REVIEW

**Comment by willdurand** [comment]
:+1:

**Comment by jbrooksuk** [comment]
Updating a components status isn't easy (pragmatically it is) but it's the mapping between an incident status and a component status that isn't.

**Comment by willdurand** [comment]
Maybe, you should leave this as is, and only mention that an incident is tied to a component in the UI, because both can be decorrelate IMO.

**Comment by jbrooksuk** [comment]
Yeah I think that Cachet should link the statuses, but what we could do is this:

1. User selects a component from the dropdown.
2. Cachet displays the inline status editor.
3. On save, the incident gets created and the component status will update too.

If no status is chosen from the inline status picker, nothing happens to it.

Thoughts?

**Comment by willdurand** [comment]
Sounds good to me. Cachet should let users:

* create incidents without any components linked
* change component statuses without having to create a new incident nor link to an existing incident

**Comment by jbrooksuk** [comment]
I'm glad you agree. As far as I'm concerned, Cachet shouldn't do anything unexpected.

In which case, I'm going to leave this PR open until I've had time to implement the inline status editor within the incident pages.

File: app/lang/en/forms.php [hunk 1]
```diff
@@ -21,6 +21,7 @@
     'incidents' => [
         'name'         => 'Name',
         'status'       => 'Status',
+        'component'    => 'Component',
```
Comment (GrahamCampbell) [inline app/lang/en/forms.php]: missing from fr

File: app/lang/en/forms.php [hunk 2]
```diff
@@ -86,5 +87,8 @@
     'delete' => 'Delete',
     'submit' => 'Submit',
     'cancel' => 'Cancel',
-    'remove' => 'Remove'
+    'remove' => 'Remove',
+
+    // Other
+    'optional' => '* Optional',
```
Comment (GrahamCampbell) [inline app/lang/en/forms.php]: missing from fr

**Pull request merged**